lib/src/drawConfigContext.dart

```diff
@@ -128,7 +127,12 @@ class DRAWConfigContext {
     _globalConfigPath = _getGlobalConfigPath();
     // Load the first file found in order of path preference.
     final primaryFile = _loadCorrectFile();
-    _customConfig = new Config.fromStrings(primaryFile.readAsLinesSync());
+    try {
+      _customConfig = new Config.fromStrings(primaryFile.readAsLinesSync());
+    } catch (exception) {
+      throw new DRAWClientError(
+          'There was an issue parsing the draw.ini file.');
```

Comment:
'Could not parse configuration file.' might be better, since we probably will want to add support for custom configuration file names.